app/src/main/java/com/nononsenseapps/feeder/ui/compose/editfeed/CreateFeedScreenViewModel.kt

```diff
@@ -81,11 +84,11 @@ class CreateFeedScreenViewModel(
             val feedId =
                 repository.saveFeed(
                     Feed(
-                        url = URL(feedUrl),
+                        url = URL(if (isNostrUri) "https://njump.me/$feedUrl" else feedUrl),
                         title = feedTitle,
                         customTitle = feedTitle,
                         tag = feedTag,
-                        fullTextByDefault = fullTextByDefault,
+                        fullTextByDefault = if (isNostrUri) false else fullTextByDefault,
```

Comment:
```suggestion
                        fullTextByDefault = fullTextByDefault,
```

Again. Do not override user's choices

Reply:
The comment for the OPMLPullParser code applies here as well.